src/components/Marketplace/MarketplaceScreen.js

```diff
@@ -1,5 +1,104 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import Airtable from 'airtable';
+import {
+  AppBar, IconButton, InputBase, Tab, Tabs, Toolbar,
+} from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import './MarketplaceScreen.css';
+import { makeStyles } from '@material-ui/core/styles';
+import FarmCard from './FarmCard';
+
+// Airtable set-up
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey }).base(airtableConfig.baseKey);
+
+// Custom styling
+const styles = makeStyles({
```

Comment:
Rename this to useStyles, and if you want to use this in the component don't forget to do something like:

const classes = useStyles();
under line 28!

Also try to use the hex colors found in the Figma